lib/runtime/src/transports/etcd/lease.rs

```diff
@@ -29,35 +35,20 @@ pub async fn create_lease(
         }
     });
 
-    Ok(Lease {
-        id,
-        cancel_token: clone,
-    })
-}
-
-/// Revoke a lease given its lease id. A wrapper over etcd_client::LeaseClient::revoke
-pub async fn revoke_lease(mut lease_client: LeaseClient, lease_id: u64) -> Result<()> {
-    match lease_client.revoke(lease_id as i64).await {
-        Ok(_) => Ok(()),
-        Err(e) => {
-            tracing::warn!("failed to revoke lease: {:?}", e);
-            Err(e.into())
-        }
-    }
```

Comment:
If I understand, we never explicitly revoke any lease, but let it either timeout or cancelled via the cancellation token on shutdown?